packages/shared/components/inputs/Pin.svelte

```diff
@@ -92,14 +91,15 @@
      * the auto-suggest feature or other event might follow
      * the keydown event and invalidate it.
      */
-    const changeHandlerHelper = (e, i: number) => {
-        if (!/^[0-9]$/.test(e.data)) {
-            inputs[i] = ''
+    function changeHandlerHelper(event, index: number): void {
```

Comment:
It doesn't work. It might be a sign that there is a bug present, but I'm not sure and I cannot test it without the mobile environment. I'm going to leave it as is and wait for mobile team to resolve it :smiling_imp:

Reply:
I think the correct type is [InputEventInit](https://w3c.github.io/uievents/#idl-inputeventinit), what you think?

Reply:
I think that's correct. I'll create a PR to develop